app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfileController < ApplicationController
+  def show
+    @profile = User.where(email: params.require(:email))
+      .select('first_name', 'last_name', 'email').first
```

Comment:
`User.find_by(email: params.require(:email))` will return the first user it finds.